web/themes/custom/pineapple/sass/pineapple.styles.scss

```diff
@@ -1,3 +1,7 @@
+// Import external libraries. Grunt-sass has been configured to have the
+// theme's npm node_modules and Bower libraries directory as an include path.
+@import "breakpoint-sass/stylesheets/breakpoint";
+
 @charset "UTF-8";
```

Comment:
This needs to go to the top of the file. There's a php storm warning.

Reply:
done